ideascube/library/static/library/library.css

```diff
@@ -39,4 +39,145 @@ img.cover {
 .book-detail h3 {
     font-family: 'fira_sansmedium';
     font-size: 1em;
-}
\ No newline at end of file
+}
+
+
+
+/* ************************************************* */
+/* ************ SECTION COLORS ********************* */
+/* ************************************************* */
+
+/************* Cartoons *************/
+#id_section > option[value=children-cartoons] {
```

Comment:
I'm not sure we can put html node into a option.
Maybe it can work with a :after/:before css rule.
I'll try.